cfpb/consumerfinance.gov
# Remove unreferenced value rule variables | rename `has_rule` to `has_bottom_rule_line`

I couldn't find any references to `has_top_border` and `has_bottom_border` outside of the template. Since they're both set on `value`, that means they should be safe to remove, yea?

## Removals

- Remove unreferenced value rule variables `has_top_border` and `has_bottom_border`.


## Changes

- Rename `has_rule` to `has_bottom_rule_line`.


## How to test this PR

1. Run migrations and visit page http://localhost:8000/admin/pages/630/edit/ and see that you can still add/remove the bottom rule line from the text introduction.

<img width="1368" alt="Screen Shot 2023-05-09 at 5 43 40 PM" src="https://github.com/cfpb/consumerfinance.gov/assets/704760/150f7cf2-1962-4fb7-94d7-2c67e0f70dc1">

<img width="1361" alt="Screen Shot 2023-05-09 at 5 43 49 PM" src="https://github.com/cfpb/consumerfinance.gov/assets/704760/ba07c773-859a-4e1d-9944-4e44be493fd8">
